### DISCUSSION TITLE
Seeking Collaborators for Web3 Learning

### DISCUSSION THREAD

**Discussion by MrJamesGold** [Cyfrin/foundry-full-course-cu#1426] [General]
Hello everyone,

My name is James, and I'm 20 years old. I am a Golang developer with almost 2 years of experience and have recently started learning Web3. I've already invested over 10 hours in this course and found it incredibly engaging and rewarding.

I have so much passion, but I find myself missing a circle of friends who share my age and interests.

I'm eager to form a small group where we can connect and talk about our findings and everything else. So, I decided to create this discussion to ask if anyone is interested.

please feel free to contact me so we can talk and get to know each other more.

Here is my telegram username: @MrJamesGold

**Comment by EngrPips** [comment]
That so thoughtful of you @MrJamesGold , They said if you want to go fast go alone but if you wan to go far go with like minds. I would say it would be very much easier to find buddies and like minds on the Cyfrin discord server. Here is the link https://discord.gg/cyfrin to join if you haven't already.
